_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -94410,6 +94375,14 @@
 /obj/effect/turf_decal/tile/neutral{
 	dir = 8
 	},
+/obj/machinery/power/apc{
+	areastring = "/area/command/heads_quarters/ce";
+	name = "Chief Engineer's APC";
+	pixel_y = 32
```

Comment:
pixel_y gets overwritten by the apc's dir, you need to add `dir = 1` . That's why it appears on the ground in the map render.

Reply:
ah ok

Reply:
ah ok ty